### PR TITLE
[8.0][project_description] New module to have project descriptions.

### PR DESCRIPTION
**Pull request by yajo**
Description is in HTML format, and quick search added too.

@rafaelbn

### CODE REVIEW

File: project_description/views/project_view.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<openerp>
+<data>
+
+<record id="project_form" model="ir.ui.view">
+    <field name="name">Project description</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.edit_project"/>
+    <field name="arch" type="xml">
+        <xpath expr="//page[@name='team']" position="before">
+            <page name="description" string="Description">
```
Comment (pedrobaeza) [inline project_description/views/project_view.xml]: name it `description_page` for being more accurate and descriptive when inheriting.

**Comment by pedrobaeza** [comment]
Minor remark, but :+1: (code review)

**Comment by rafaelbn** [comment]
Runbot fails

**Comment by pedrobaeza** [comment]
It didn't fail. It's just that it has been killed because probably the server has been rebooted. I have forced the rebuild.

File: project_description/README.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline project_description/README.rst]: https

**Comment by elicoidal** [comment]
LGTM too

File: project_description/views/project_view.xml (hunk 1 above)
Comment (yajo) [inline project_description/views/project_view.xml]: Isn't `//page[@name='description']` explicit enough?

Comment (yajo) [inline project_description/views/project_view.xml]: Nevermind, changed.

**Comment by dreispt** [comment]
:+1:

**Pull request merged**